internal/cypress/suite/suite.go

```diff
@@ -35,8 +35,10 @@ func SortByHistory(suites []Suite, history insights.JobHistory) []Suite {
 			delete(hash, s.Name)
 		}
 	}
-	for _, v := range hash {
-		res = append(res, v)
+	for _, v := range suites {
+		if _, ok := hash[v.Name]; ok {
+			res = append(res, v)
+		}
```

Comment:
If this order is important, it should probably be considered as part of the tests too?

Reply:
yup. The test verifies the order https://github.com/saucelabs/saucectl/blob/main/internal/cypress/suite/suite_test.go#L56. Due to the unordered map, the unit test failed occasionally. After this fix, the test will pass constantly.

Reply:
Sweet!